code/modules/mod/modules/modules_antag.dm

```diff
@@ -494,3 +494,39 @@
 	if(deleting)
 		return
 	mod.helmet.flash_protect = initial(mod.helmet.flash_protect)
+/obj/item/mod/module/minigun
+	name = "MOD laser gatling gun module"
+	icon_state = "minigun_module"
+	inhand_icon_state = "minigun"
+	desc = "A laser gatling gun with a that connects to a MOD suit"
+	w_class = WEIGHT_CLASS_HUGE
+	idle_power_cost = DEFAULT_CHARGE_DRAIN * 0.5
+	use_power_cost = DEFAULT_CHARGE_DRAIN * 3
+	var/overheat = 0
+	var/overheat_max = 30
+	var/heat_diffusion = 0.5
```

Comment:
move these below the vars that are inherited (not having a `var/` in front of them)